src/tir/transforms/ir_utils.h

```diff
@@ -267,6 +267,24 @@ class ConditionalBoundsContext {
   std::unordered_map<const VarNode*, arith::IntSet> origin_map_;
 };
 
+// Information of tensor core fragment.
```

Comment:
Maybe we should consider some further refactoring and generalization, but I agree it's way above the scope of this PR for now